README.md

```diff
@@ -13,33 +13,30 @@
 
 # PyGridSim
 
-Package to simulate OpenDSS circuits on Python.
-
-NOTE: README comes from dai cookie cutter, will be updated
-NOTE: will be moved to Dai lab repository
+PyGridSim is a package that allows user to simulating OpenDSS circuits with Python. PyGridSim uses a functional interface to allow users to efficiently generate circuits of various scopes.
 
 - Documentation: https://amzhao.github.io/PyGridSim
 - Homepage: https://github.com/amzhao/PyGridSim
 
 # Overview
 
-PyGridSim aims to provide accessible access to tools like OpenDSS, AltDSS using Python. The goal is to create large-scale electrical circuits representing residential neighborhoods (and other scenarios) using an intuitive interface, without any background in OpenDSS software.
+PyGridSim allows user to create circuits with the amount of customization they desire. Thus, users can either fully specify each component they add to the circuit, or lean on library-provided parameter sets. PyGridSim supports the batch creation of every circuit component, emphasizing scalability and efficiently in building large circuits.
```

Comment:
This description is a bit formal. I would simplify it:

PyGridSim allows users to create and customize circuits. Users can either fully specify each component they add to the circuit, or lean on library-provided parameter sets. PyGridSim supports the batch creation of every circuit component, emphasizing scalability and efficiently in building large circuits.